stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/cookbook/ui/ChannelList.kt

```diff
@@ -43,7 +43,7 @@ class ChannelList : Fragment() {
         val channelListFactory: ChannelListViewModelFactory = ChannelListViewModelFactory(
             filter = Filters.and(
                 Filters.eq("type", "messaging"),
-                Filters.`in`("members", listOf(ChatDomain.instance().currentUser.id)),
+                Filters.`in`("members", ChatDomain.instance().user.value?.let(::listOf) ?: emptyList()),
```

Comment:
🤔 This will create different behavior when the user is not set. I think we shouldn't do that in documentation

Reply:
We can use `!!` or smart cast in a variable outside the factory. What do you think?

Reply:
I think we can use `!!` for now. Did you also update CMS and docusaurus samples?

Reply:
Ops, no. I'll that